assets/styles/main.scss

```diff
@@ -32,7 +32,7 @@
 
 // Vendors
 // ==========================================================================
-@import "node_modules/locomotive-scroll/dist/locomotive-scroll";
+@import "../../node_modules/locomotive-scroll/dist/locomotive-scroll";
```

Comment:
Was there a warning from Sass for this change?

Reply:
![image](https://github.com/user-attachments/assets/99bab13f-0da1-402e-86e4-6eb58a02b8f9)
Yeah this is the error i'm receiving without the tweaked path.
It seems to work correctly when importing sass files, but breaks for css files (which is the case in this instance)

Reply:
Hrm. Must be a difference between `sass.render` and `sass.compile`?